src/Octoshift/Commands/ReclaimMannequin/ReclaimMannequinCommandHandler.cs

```diff
@@ -36,10 +38,21 @@ public async Task Handle(ReclaimMannequinCommandArgs args)
                 throw new OctoshiftCliException($"File {args.Csv} does not exist.");
             }
 
-            await _reclaimService.ReclaimMannequins(GetFileContent(args.Csv), args.GithubOrg, args.Force);
+            //TODO: Get verbiage approved
+            if (args.SkipInvitation)
+            {
+                _ = _confirmationService.AskForConfirmation("Reclaiming mannequins with the --skip-invitation option is immediate and irreversible. Are you sure you wish to continue? (y/n)");
+            }
+
+            await _reclaimService.ReclaimMannequins(GetFileContent(args.Csv), args.GithubOrg, args.Force, args.SkipInvitation);
         }
         else
         {
+            if (args.SkipInvitation)
+            {
+                throw new OctoshiftCliException($"--csv must be specified to skip reclaimation email");
+            }
```

Comment:
I suggest moving this validation to the (new) ReclaimMannequinCommandArgs.Validate() method.

Reply:
👍🏾 I can refactor as part of the follow-up PR, was hoping to get this out since it has been open for a while and I have finally got the functional test to pass.